.github/workflows/delete-gh-pages-folder.yml

```diff
@@ -1,20 +1,25 @@
-name: Delete gh-pages folder
+name: Delete build folder on branch deletion
 
-on:
-  delete: branches
+on: delete
 
 jobs:
   delete-folder:
     runs-on: ubuntu-latest
     steps:
+      - name: Extract branch name
+        shell: bash
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_HEAD_REF#refs/heads/})"
+        id: extract_branch
+
       - name: Checkout
         uses: actions/checkout@v3
         with:
           repository: mi6/ic-design-system-githubpages
 
       - name: Remove and commit
```

Comment:
Can we add a commit message to this step so we have a record of what's getting deleted?

Reply:
Have added a commit message